src/swift/flwr/Sources/Flower/Common/Parameter.swift

```diff
@@ -0,0 +1,214 @@
+//
+//  Parameter.swift
+//  FlowerSDK
+//
+//  Created by Daniel Nugraha on 09.01.22.
+//
+
+import Foundation
+import NumPySupport
+import PythonSupport
+import PythonKit
+import CoreML
+import NIOCore
+import NIOPosix
+
+let appDirectory = FileManager.default.urls(for: .applicationSupportDirectory, in: .userDomainMask).first!
+
+@available(iOS 14.0, *)
+public class ParameterConverter {
+    private var np: PythonObject?
+    
+    /// The permanent location of the numpyArray.
+    private var numpyArrayUrl = appDirectory.appendingPathComponent("numpyArray.npy")
+    private var group: EventLoopGroup?
+    
+    public static let shared = ParameterConverter()
+    
+    private init() {
+        initGroup()
+    }
+    
+    private func initGroup() {
+        if group == nil {
+            print("Opening Python event loop group")
```

Comment:
We generally avoid using print statements in Python, and I think this would be desirable in the Swift SDK too. Do you think it would be much effort to use, e.g. https://developer.apple.com/documentation/os/logger, with proper log levels so, e.g. a caught error is logged with `logger.error`?